DOWNLOAD_SNAPSHOT_JAR.md

```diff
@@ -0,0 +1,7 @@
+# Instructions to download the latest snapshot version of the validator
+
+The GitHub Action [`test_package.yml`](../.github/workflows/test_package.yml) is responsible for automatically deploying the project when a push is made to the `master` branch. The artifacts can be found on GitHub Packages: [GTFS Realtime Validator Packages](https://github.com/orgs/MobilityData/packages?repo_name=gtfs-realtime-validator).
```

Comment:
Now that this page is significantly simplified I'd remove it entirely, and just put the links to the library and webapp packages in-line in the respectively sections of the README, just like they were before. Looks like you've added the links, and I don't think this file is referenced anywhere else, so you can delete it.